src/coverlet.console/Program.cs

```diff
@@ -14,6 +15,8 @@
 
 namespace Coverlet.Console
 {
+    using ThresholdTypeFlagValues = Dictionary<ThresholdTypeFlags, double>;
```

Comment:
personally I don't like alias in this case, I prefer clear dic intantiation, I think is more clear.

Reply:
Updated Type usage in all places.